openlibrary/plugins/upstream/merge_authors.py

```diff
@@ -85,31 +126,26 @@ def merge_property(self, a, b):
         else:
             return a
 
-    def make_redirect_doc(self, key, redirect):
-        return {
-            "key": key,
-            "type": {"key": "/type/redirect"},
-            "location": redirect
+
+class AuthorRedirectEngine(BasicRedirectEngine):
+    def find_references(self, key):
+        q = {
+            "type": "/type/edition",
+            "authors": key,
```

Comment:
Author merging does not update edition references (because the `"authors": key` appears to be wrong, likely should be `"authors": {"author": {"key": key}},` as shown below